score/player.c

```diff
@@ -10,22 +14,35 @@ void player_init(struct Player* player)
 
     player->highScore = 0;
 }
-
+/**
+ * @param currPlayer return values of player initial array 
```

Comment:
This @param should describe what the `currPlayer` parameter is, then below that you can do @return to describe what the return value represents